webhook/workspace/handler/immutable.go

```diff
@@ -33,6 +33,9 @@ var ImmutableWorkspaceDiffOptions = []cmp.Option{
 	// field managed by cluster and should be ignored while comparing
 	cmpopts.IgnoreFields(metav1.ObjectMeta{}, "ManagedFields", "Finalizers", "DeletionTimestamp"),
 	cmpopts.IgnoreFields(devworkspace.DevWorkspaceSpec{}, "Started"),
+	cmpopts.IgnoreMapEntries(func(k string, v string) bool {
+		return k == config.StoppedByAnnotation
```

Comment:
does it mean that it will be ignored in all possible maps? Labels, annotations, devfile attributes?

There is no security leak, but that's not accurate check.
I'm fully OK with proceeding with it, but we must consider moving to production-ready approach, which Angel proposed some time ago

Reply:
Now I checked the master branch, and it's just the same, apart from formatting )